### ISSUE TITLE
remote sync multiple files?

### ISSUE THREAD

**Issue by okeniiawak** [yongkangchen/remote-sync#265]
I **can** select entire project folder and sync everything
I **can** select any folder and upload with its contents
I **can** select a single file and sync it
I **cannot** select multiple files and sync them

Sometimes it is important to do it because remote-sync has to do all over again if you forget something.
Am I missing something? I am talking about the context menu with an option to remote sync.

**Comment by Psychosynthesis** [comment]
Yes, I really really need to be able to put a watcher on the entire folder, because there are so many files in there!

P.s. Is project still updates sometimes?

**Comment by mbalandis** [comment]
@Psychosynthesis way to revive a zombie thread, haha! 💀

**Comment by Psychosynthesis** [comment]
@mbalandis I cherish hope
